src/tools/schema2wasm-utils.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+// https://kotlinlang.org/docs/reference/keyword-reference.html
+// [...document.getElementsByTagName('code')].map(x => x.innerHTML);
+// Includes reserve words for Entity Interface
+export const kotlinKeywords = [
+    'as', 'as?', 'break', 'class', 'continue', 'do', 'else', 'false', 'for', 'fun', 'if', 'in', '!in', 'interface', 'is',
+    '!is', 'null', 'object', 'package', 'return', 'super', 'this', 'throw', 'true', 'try', 'typealias', 'val', 'var',
+    'when', 'while', 'by', 'catch', 'constructor', 'delegate', 'dynamic', 'field', 'file', 'finally', 'get', 'import',
```

Comment:
FWIW only the words between 'as' and 'while' are hard keywords. The rest can be used as identifiers, still, that's not what your change is about.